nimterop/exprparser.nim

```diff
@@ -45,14 +45,14 @@ proc getExprIdent*(gState: State, identName: string, kind = nskConst, parent = "
   ##
   ## Returns PNode(nkNone) if the identifier is blank
   result = newNode(nkNone)
-  if identName notin gState.skippedSyms:
+  if gState.currentExprSkipIdentValidation or identName notin gState.skippedSyms:
     var ident = identName
     if ident != "_":
       # Process the identifier through cPlugin
       ident = gState.getIdentifier(ident, kind, parent)
     if kind == nskType:
```

Comment:
Why isn't this being checked first in the if block?

Reply:
Because we still want the ident to be parsed if `kind = nskConst` and having the check in the first if block would skip that

Reply:
Oh okay, I didn't see the elif is in the same block, not parent.